src/api/public/apidocs-new/paths/source_project_name.yaml

```diff
@@ -0,0 +1,23 @@
+post:
+  summary: Add channels to a source
```

Comment:
This is only true if the command is `addchannels`. Otherwise, this endpoint does something completely different.

Reply:
Yeah, true

Reply:
I've changed the apidoc so it makes reference only to this command, I think it makes much sense now.